### ISSUE TITLE
Installing Expo client on iOS simulator not installing

### ISSUE THREAD

**Issue by glasje** [expo/expo-cli#933]
### Environment
run expo start --ios

### Steps to Reproduce

when I execute the project with expo start --ios, the application is lifted but it does not install the expo in the simulator, I already tried with models iphone 6,7,8, x, xr, xs and still there is no change. The same happens if I run the app with react-native run-ios does not install the app.
version S.O. : mojave 10.14.3
version expo-cli: 3.0.9
version xcode: Version 10.3 (10G8)
version simulator: Version 10.3 (SimulatorApp-880.5 CoreSimulator-587.35)
### Reproducible Demo
![image](https://user-images.githubusercontent.com/34929785/62892161-91d30780-bd15-11e9-966a-53966fd28a01.png)

**Comment by glasje** [comment]
@cruzach  It happens to me with blank projects, with developed app and we install expo cli in the simulator.
I did tests on a co-worker's mac and exactly the same thing happens

**Comment by cruzach** [comment]
What do you mean by "we installed expo cli in the simulator"? Can you try running `expo client:install:ios`

**Comment by glasje** [comment]
I ran the indicated command and threw this after execution.
![image](https://user-images.githubusercontent.com/34929785/63056499-32602d80-beb6-11e9-84dc-14b6c557de6a.png)
 
I need to solve this issue urgently. :(

**Comment by cruzach** [comment]
Could you try reinstalling [`watchman`](https://docs.expo.io/versions/latest/introduction/installation/#watchman)? Also, go ahead and share your `app.json` here

**Comment by glasje** [comment]
if you reinstall watchman but the same error came out
watchman version : 4.9.0
this is app.json :
{
  "expo": {
    "name": "prueba",
    "slug": "AwesomeProject",
    "privacy": "public",
    "sdkVersion": "34.0.0",
    "platforms": [
      "ios",
      "android",
      "web"
    ],
    "version": "1.0.0",
    "orientation": "portrait",
    "icon": "./assets/images/icon.png",
    "splash": {
      "image": "./assets/images/splash.png",
      "resizeMode": "contain",
      "backgroundColor": "#ffffff"
    },
    "updates": {
      "fallbackToCacheTimeout": 0
    },
    "assetBundlePatterns": [
      "**/*"
    ],
    "ios": {
      "supportsTablet": true
    }
  }
}
-------------------------------
this is package.json
{
  "main": "node_modules/expo/AppEntry.js",
  "scripts": {
    "start": "expo start",
    "android": "expo start --android",
    "ios": "expo start --ios",
    "web": "expo start --web",
    "eject": "expo eject",
    "test": "jest --watchAll"
  },
  "jest": {
    "preset": "jest-expo"
  },
  "dependencies": {
    "@expo/samples": "~3.0.3",
    "@expo/vector-icons": "^10.0.3",
    "@react-navigation/web": "^1.0.0-alpha.9",
    "expo": "^34.0.1",
    "expo-asset": "^6.0.0",
    "expo-constants": "6.0.0",
    "expo-font": "~6.0.0",
    "expo-web-browser": "6.0.0",
    "react": "16.8.3",
    "react-dom": "^16.8.6",
    "react-native": "https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz",
    "react-native-gesture-handler": "~1.3.0",
    "react-native-web": "^0.11.4",
    "react-navigation": "^3.11.0"
  },
  "devDependencies": {
    "babel-preset-expo": "^6.0.0",
    "jest-expo": "^34.0.0"
  },
  "private": true
}

**Comment by JeremyMeissner** [comment]
Hi I have the same problem except the Expo Client app is installed but the app is not starting :/
<img width="1080" alt="Capture d’écran 2019-08-22 à 13 52 05" src="https://user-images.githubusercontent.com/34174370/63512442-1f67e180-c4e4-11e9-8fe5-5f555102598e.png">

**Comment by cruzach** [comment]
@JeremyMeissner what version of the Expo Client, what expo-cli version (run `expo diagnostics`), and what if you try it on a different simulator?

**Comment by JeremyMeissner** [comment]
I tested on the iPhone 8 Plus and the iPhone 8 and I have the same problem @cruzach :/

the expo diagnostics return:
```
Expo CLI 3.0.10 environment info:
    System:
      OS: macOS 10.14.6
      Shell: 3.2.57 - /bin/bash
    Binaries:
      Node: 10.16.2 - /usr/local/bin/node
      npm: 6.10.3 - /usr/local/bin/npm
      Watchman: 4.9.0 - /usr/local/bin/watchman
    IDEs:
      Xcode: 11.0/11M336w - /usr/bin/xcodebuild
    npmPackages:

      expo: ^34.0.1 => 34.0.4
      react: 16.8.3 => 16.8.3
      react-native: https://github.com/expo/react-native/archive/sdk-34.0.0.tar.gz => 0.59.8
    npmGlobalPackages:
      expo-cli: 3.0.10
```

**Comment by cruzach** [comment]
What iOS are the sims? Wondering if this due to something specific with Xcode 11 beta?

**Comment by JeremyMeissner** [comment]
The simulations was on IOS 13.0, I choose the beta because on this docs: https://docs.expo.io/versions/latest/workflow/ios-simulator/
<img width="1118" alt="Capture d’écran 2019-08-22 à 16 53 37" src="https://user-images.githubusercontent.com/34174370/63525289-b9885380-c4fd-11e9-8009-09c63daf85f3.png">
They say: 
> selecting the most recent version
And for me the most recent is the beta
<img width="1440" alt="Capture d’écran 2019-08-22 à 16 54 40" src="https://user-images.githubusercontent.com/34174370/63525353-d9b81280-c4fd-11e9-95a9-3262ed245182.png">

**Comment by JeremyMeissner** [comment]
You right, the beta is broken, I always accidentally open the Simulator Beta instead of just Simulator.
I switch to Xcode 10.3 and open Simulator on a 12.4 device and all work, thanks you !

**Comment by mohammadou1** [comment]
Am have the same issue but with expo icon being installed on the simulator :( can't revert back to xcode 10.3 and iOS 13 isnt beta anymore
did anyone solve it

**Comment by josermorales** [comment]
> Hi I have the same problem except the Expo Client app is installed but the app is not starting :/
> <img alt="Capture d’écran 2019-08-22 à 13 52 05" width="1080" src="https://user-images.githubusercontent.com/34174370/63512442-1f67e180-c4e4-11e9-8fe5-5f555102598e.png">

same :/

**Comment by josermorales** [comment]
@mohammadou1 It looks like @cruzach  solved it in #957 we just have to wait for the changes to be merged

**Comment by kenardrs** [comment]
Same problem here... the app never opens even though expo seems to be installed on the device ... how can i revert xcode back to 10.0.3?

**Comment by DatMoser** [comment]
Same here, only way to work around it is to open the app and add the packager address manually. Hope to get fixed asap!

**Comment by kenardrs** [comment]
It works!!!! @cruzach solve it in https://github.com/expo/expo-cli/pull/957... i just change the file Simulator.js and it works fine... thanks bro...

**Comment by josermorales** [comment]
@kenardrs I was trying to find that file to fix it manually but couldn’t find it. Send help <3

**Comment by quowtf** [comment]
> @kenardrs I was trying to find that file to fix it manually but couldn’t find it. Send help <3

What I did and be able to start working was, add manually the lan url to expo client: `exp://192...`
![image](https://user-images.githubusercontent.com/3241622/65380748-ba9cd400-dca7-11e9-85cd-d89a1977033a.png)

**Comment by kenardrs** [comment]
> @kenardrs I was trying to find that file to fix it manually but couldn’t find it. Send help <3

@josermorales, In my case the path to the file was '/usr/local/lib/node_modules/expo-cli/node_modules/@expo/xdl/build/Simulator.js', consider i am using default installation of expo-cli to mac os. If you have some issues to find it in your own system... just use the 'find' command with root privilegies on a terminal like this 'sudo find / -name Simulator.js'. Edit the file, also using root privilegies, as suggested by @cruzach and have fun :)

**Comment by komalsikka** [comment]
I got two options to work this out:

1. On Xcode 11, it got stuck after installing expo, but app didn't install. 
Work around: Open expo, click on the + sign on the top right and enter the URL of the app manually. It works perfectly and syncs real-time.

2. I downgraded to Xcode 10.3 (latest stable version) and iOS 12 simulator. Expo installed and process got hung after that. Open expo manually and the hung process moved forward, app installed and opened. This happened only for the first time. Subsequent runs worked like a charm.
